src/tools/scripts/populate-zrr.js

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env node
+'use strict';
+
+const CSVToJSON = require('csvtojson');
+const { execute } = require('../utils');
+const { program } = require('commander');
+
+program
+.option('-c, --csv <path>', 'CSV file path');
```

Comment:
@jystervinou est-ce qu'on rajoute le helpOption ?
`program.helpOption('-e', 'HELP command');`

Reply:
si on ne met rien ça n'en crache pas un par défaut à partir des .option ?

Reply:
https://github.com/tj/commander.js/#automated-help

Reply:
@jystervinou il me semble que non, quand j'avais fait le test avec createUser.js, il mettait juste un error: Paramètres invalides

Reply:
node src/tools/scripts/populate-zrr.js --help
Usage: populate-zrr [options]

Options:
  -c, --csv <path>  CSV file path
  -h, --help        display help for command

Reply:
Nickel !